waterlog/src/Components/MapFullScreen.js

```diff
@@ -107,78 +89,50 @@ class MapFullScreenComponent extends Component {
             </button>
           </div>
           <div className={`map-icon-div-layer2-fullscreen map-button-tab ${this.state.simpleView ? "" : "invisible"}`}>
-            
+
             <img
-              className="icon"
+              className="icon-home"
+              src={require("../images/more_map_icon.png")}
```

Comment:
note previous comment on icons and variables

Reply:
implemented